src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,115 @@
+
+.cards {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  @media screen and (width:1200px) {
```

Comment:
Use 'max-width' instead of 'width' in the media query to target devices with a maximum width of 1200px.